ecs-agent/api/attachment/resource/ebs_discovery_windows.go

```diff
@@ -80,8 +80,8 @@ func parseExecutableOutput(output []byte, candidateVolumeId string, candidateDev
 				"Output:%s", candidateVolumeId, candidateDeviceName, out)
```

Comment:
nit: is `candidateDeviceName` still needed here?

Reply:
No, it's only used for the error message which is already taken care of in L47. I have removed this in the latest commit, thanks!